README.md

```diff
@@ -34,6 +34,15 @@ graph-sbol -i my_file.ttl
 
 The `excel-to-sbol` utility reads an Excel file specifying a library of basic and composite parts, formatted following `sbol_library_template.xlsx`.
 
+### Convert beween SBOL3 and other genetic design formats
+
+The `sbol-converter` utility converts between any of the SBOL3, SBOL2, GenBank, and FASTA formats.
+
+Additional "macro" utilities convert specifically between SBOL3 and one of the other formats: 
+- `sbol2fasta` and `fasta2sbol` convert to and from FASTA
+- `sbol2genbank` and `genbank2sbol` convert to and from GenBank
```

Comment:
It looks like these convert between SBOL 3 and FASTA, but the name `sbol2genbank` sort of looks like it might do SBOL2 conversion. It might be useful to explicitly state the conversion is between SBOL 3 and FASTA.

Reply:
Done.